content/event-platform/available-events.md

```diff
@@ -1139,7 +1139,7 @@ The event is triggered when any changes are made to the record values.
 
 > Example of payload
 
-```json
+```json [snippet:Payload]
```

Comment:
this will be overwritten the next time we generate this doc.

maybe adapt the script which is in PIM repository

Reply:
Solved in https://github.com/akeneo/pim-enterprise-dev/pull/25291